### ISSUE TITLE
[BUG] CHIP Error 172

### ISSUE THREAD

**Issue by iVoGia** [project-chip/connectedhomeip#33727]
### Reproduction steps

Step:
1/ Scan QR. 
2/ Start pairing device.

### Bug prevalence

Newbie

### GitHub hash of the SDK that was being used

matter device

### Platform

darwin, other

### Anything else?

I'm following the struct guide: https://developer.apple.com/documentation/mattersupport/adding-matter-support-to-your-ecosystem
I met the error as below. 
Pls take a look at below image. 
Pls help
![Screenshot 2024-06-04 at 16 57 16](https://github.com/project-chip/connectedhomeip/assets/39343021/c84af7ad-f9bf-48e8-b81c-83fb803a443a)
 me 
Thanks

**Comment by bzbarsky-apple** [comment]
@iVoGia That looks like the device has its certs signed by a PAA that is not known to the commissioner (101 during device attestation is kPaaNotFound).

What PAA is being used here?

**Comment by bzbarsky-apple** [comment]
And what list of PAAs did you provide when starting the controller or controller factory (depending on which exact controller startup API you are using)?

**Comment by iVoGia** [comment]
Hi @bzbarsky-apple , 
The factory defined as below:
![Uploading Screenshot 2024-06-05 at 08.01.59.png…]()

**Comment by iVoGia** [comment]
> @iVoGia That looks like the device has its certs signed by a PAA that is not known to the commissioner (101 during device attestation is kPaaNotFound).
> 
> What PAA is being used here?

What's the PAA, can you explain for me ? 
The device is Start Lightbulb.

**Comment by iVoGia** [comment]
How to I can get device list from Matter Accessories in the Setting ? 
Get all from Fabric ?

**Comment by bzbarsky-apple** [comment]
OK, so in the factory params you are leaving `productAttestationAuthorityCertificates` as nil.  The docs say:
```
/*
 * The Product Attestation Authority certificates that are trusted to sign
 * device attestation information (and in particular to sign Product Attestation
 * Intermediate certificates, which then sign Device Attestation Certificates).
 *
 * Defaults to nil.
 */
```

So if you leave that nil, it will fall back to the test-only PAA being trusted, so you will not be able to verify attestation on any production devices.

> What's the PAA, can you explain for me ?

See document comment above, or the Matter specification, but basically it's a certificate signing root for attestation certificates.

The intent is that you select which PAAs you trust (possibly all the ones in the DCL, but it's up to you; it's a policy decision).

See also documentation at https://developer.apple.com/documentation/matter/onboarding_a_matter_device which says, as one of the things you need to do:

> Set the factory parameter object’s [productAttestationAuthorityCertificates](https://developer.apple.com/documentation/matter/mtrdevicecontrollerfactoryparams/4122941-productattestationauthoritycerti) to an array of trusted Matter certificates.

**Comment by iVoGia** [comment]
Hi @bzbarsky-apple  ,
Thank for your explained. I got it. 
I passed the paa cert for testing, and continues commission device without PAA in the AttestationDelegate. 
And now, I can commission device successfull for test. 
Pls take a look at below image. 
![IMG_4486](https://github.com/project-chip/connectedhomeip/assets/39343021/20c64674-79a1-43de-b2bd-da0b4822313f)
And I can establish a connection to device in a RequestHandle.swift. 
But, when I call MTRGetConnectedDevice method , it’s always return error: “ cannot establish a connection with timeout.”
Pls help me for above error. 
Thanks.
